sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -74,28 +72,11 @@
   private static class NotSerializableClass { }
 
   @Test
-  public void testSerializableFallbackCoderProvider() throws Exception {
-    CoderRegistry registry = CoderRegistry.createDefault();
-    registry.setFallbackCoderProvider(SerializableCoder.PROVIDER);
-    Coder<?> serializableCoder = registry.getDefaultCoder(SerializableClass.class);
-
-    assertEquals(serializableCoder, SerializableCoder.of(SerializableClass.class));
-  }
-
-  @Test
-  public void testAvroFallbackCoderProvider() throws Exception {
```

Comment:
It will crash in the middle of generating a schema, right? Or, it might return a nondeterministic coder which will later fail at a GBK. But if it returns a coder that just doesn't work, that is a huge problem.